### DISCUSSION TITLE
[ADDED] Scaling size slider to main tab :)

### DISCUSSION THREAD

**Discussion by 1001celeritas** [upscayl/upscayl#753] [Ideas]
Hey this would make sense to me, to make this choice part of the Main 'Upscayl'  tab and remove it from 'Settings' .

Love the software many thanks

**Comment by NayamAmarshe** [comment]
For sure!

**Comment by 1001celeritas** [comment]
🙂
i love this thing you made, its useful and looks pretty damn good to boot!
i've been passing images through it just for the effect and then blending in trad resized images to re-establish  some grain.
I use comfyUI (you must know it 🙂and wanted something simple to use just for upscayl
you hit the spot!
i'm mind blown man, upscaling can now add something to an image.
nice to talk to you sir.
ty so much for making this...

john

________________________________

**Comment by 1001celeritas** [comment]
Hey Nayam, I do get an occasional error with the very long filenames produced by some ai digital art applications i'm using windows 11. see attached image
thx again

________________________________

**Reply by aaronliu0130** [reply]
github's email discard attachments, please attach them on github instead of sending email

also, that sounds like a windows limitation. maybe we could truncate them

**Reply by 1001celeritas** [reply]
Hey thanks for your reply and for the changes to upscayl in v2.11.

I jiggled about with your main side menu, in gimp so that, when upscayl is viewed on a 1080p screen with a standard windows task bar, the menu fits on the screen and no scrollbar appears.

see attached... 
![Screenshot 2024-05-01 085137](https://github.com/upscayl/upscayl/assets/113988154/6c1201a4-fef2-4c9a-9ff8-a427eb03f969)
![UPSCAYL](https://github.com/upscayl/upscayl/assets/113988154/90ab01e7-cf2c-442d-a4f2-5146f4343a09)

it looks a bit neater, ty again, 

i love upscayl :)

**Reply by aaronliu0130** [reply]
@NayamAmarshe I think that this would entail adding a CSS rule for the final button only to not have padding/margins below it
